bufferapp/app-shell
# [FIX] Check canEdit before opening planSelector

JIRA: [GROW-429](https://buffer.atlassian.net/browse/GROW-429)

How to test:
1. Run appshell locally
2. Visit `https://appshell.local.buffer.com:3000/#planSelector` with a admin user who has a verified email and canManageBilling set to true.
3. The planSelector should open.

Test a user who does not have permission to manage billing:
1. Run appshell locally
2. Visit `https://appshell.local.buffer.com:3000/#planSelector` with a memeber user who does not have a verified email or their canManageBilling is set to false
3. The planSelector should NOT open.

Before:
<img width="1239" alt="Screen Shot 2022-04-14 at 10 30 21 am" src="https://user-images.githubusercontent.com/7763710/163305085-4ef5ec78-7b65-4fc9-8ed7-b85f26a5b744.png">


After:
<img width="1253" alt="Screen Shot 2022-04-14 at 10 30 41 am" src="https://user-images.githubusercontent.com/7763710/163305104-37ea62c1-a615-46fe-ab33-869e4a82a827.png">
